docs/pipeline.rst

```diff
@@ -421,13 +434,17 @@ intent_classifier_tensorflow_embedding
     It is recommended to use ``intent_featurizer_count_vectors`` that can be optionally preceded
     by ``nlp_spacy`` and ``tokenizer_spacy``.
 
+    .. note:: If during prediction time a message contains **only** words unseen during training,
+              and no OOV preprocessor was used,
+              empty intent ``""`` is predicted with confidence ``0.0``.
```

Comment:
`None` is not handled by sklearn or smth

Reply:
what do you need sklearn for, you already got a classifier here? not sure what you mean

Reply:
If I remember correctly when evaluate script get called, it uses sklearn, which complains if there is a `None`, because it cannot compare string to `None`

Reply:
Yes, in `evaluate.py` : `TypeError: '<' not supported between instances of 'str' and 'NoneType'`

Reply:
yes but let's change https://github.com/RasaHQ/rasa_nlu/blob/master/rasa_nlu/evaluate.py#L131 to check for `None` as well then. I think it is better to return `None` as a string than the empty string.

Reply:
yes please